exercises/02 - Publish-Subscribe.md

```diff
@@ -206,7 +206,7 @@ In the `Divergent.Finance` project create the `InitiatePaymentProcessCommandHand
 ```c#
 namespace Divergent.Finance.Handlers
 {
-    class InitiatePaymentProcessCommandHandler : IHandleMessages<InitiatePaymentProcessCommand>
+    public class InitiatePaymentProcessCommandHandler : IHandleMessages<InitiatePaymentProcessCommand>
```

Comment:
👍 but we need to update the "after" solution to match this: https://github.com/Particular/Workshop.Microservices/blob/074f66c1a25353be73c955a1fa0290e29b3c778d/exercises/src/02%20Publish-Subscribe/after/Divergent.Finance/Handlers/InitiatePaymentProcessCommandHandler.cs#L10